README.md

```diff
@@ -0,0 +1,41 @@
+# `iiif` formula
+
+This repository contains instructions for installing and configuring the `iiif`
+project.
+
+This repository should be structured as any Saltstack formula should, but it 
+should also conform to the structure required by the [builder](https://github.com/elifesciences/builder) 
+project.
+
+## Related:
+
+* eLife's loris-docker fork: https://github.com/elifesciences/loris-docker
```

Comment:
Would make clear that this is not a fork of the project, just of a random repository that was dockerizing it.

Reply:
Ok not a random repository, the official one that was dockerizing it.